src/sider/oppsummering/sykmeldt-oppsummering-besvarelser.less

```diff
@@ -19,10 +19,15 @@
     }
```

Comment:
Kan stylingen her og i ordinaer-oppsummering-besvarelser.less slås sammen? Dvs bruke samme klasser/styling for listen